sdk/go/x/auto/local_workspace_test.go

```diff
@@ -31,21 +31,21 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const pulumiOrg = "moolumi"
+const pulumiOrg = "pulumi-test"
```

Comment:
we should use whoami or make this an envvar versus hard coding it

Reply:
this is the permanent engineering test account, it shouldn't change any more. Or are there other reasons for turning it into an env?

Reply:
The primary reasons are to decouple the code from the CI env e.g. we've had projects start out in `moolumi` and move around since that became unmanageable for all repos.

Reply:
cc @stack72 for input ^

Reply:
Ah, this is definitely a good idea for anyone who wants to build the project and run tests locally. I've gone ahead and updated to use `PULUMI_TEST_ORG` and default to "pulumi-test" if the env isn't set.